tox.ini

```diff
@@ -1,26 +1,29 @@
 [tox]
-envlist = py26, py27, py33, py34, py35, flake8
-
-[travis]
-python =
-    3.5: py35
-    3.4: py34
-    3.3: py33
-    2.7: py27
-    2.6: py26
-
-[testenv:flake8]
-basepython=python
-deps=flake8
-commands=flake8 pystac
+envlist = py34, py35, py36
 
 [testenv]
-setenv =
-    PYTHONPATH = {toxinidir}
+basepython =
+    py34: python3.4
+    py35: python3.5
+    py36: python3.6
+
+commands =
```

Comment:
This should be:

```
commands =
    check-manifest --ignore tox.ini
    {envpython} setup.py check -m -r -s
    # flake8 .
    {envpython} setup.py test
```

To prevent an issue with Flake8 being disabled, I think it would be good to create an issue to fix it later.